xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -569,34 +580,39 @@ void DX::DeviceResources::ResizeBuffers()
   else
   {
     // Otherwise, create a new one using the same adapter as the existing Direct3D device.
-    DXGI_SWAP_CHAIN_DESC1 swapChainDesc = { 0 };
+    DXGI_SWAP_CHAIN_DESC1 swapChainDesc = {};
     swapChainDesc.Width = lround(m_outputSize.Width);
     swapChainDesc.Height = lround(m_outputSize.Height);
     swapChainDesc.Format = DXGI_FORMAT_B8G8R8A8_UNORM;
     swapChainDesc.Stereo = bHWStereoEnabled;
     swapChainDesc.BufferUsage = DXGI_USAGE_RENDER_TARGET_OUTPUT;
-    swapChainDesc.BufferCount = 3 * (1 + bHWStereoEnabled);
-    swapChainDesc.SwapEffect = DXGI_SWAP_EFFECT_FLIP_SEQUENTIAL;
+    // HDR 60 fps needs 6 buffers to avoid frame drops but in Windows 10 it's good for all
+    swapChainDesc.BufferCount =
+        (m_d3dFeatureLevel >= D3D_FEATURE_LEVEL_12_0) ? 6 : 3 * (1 + bHWStereoEnabled);
+    swapChainDesc.SwapEffect = (m_d3dFeatureLevel >= D3D_FEATURE_LEVEL_12_0)
+                                   ? DXGI_SWAP_EFFECT_FLIP_DISCARD
```

Comment:
why used `FLIP_DISCARD` for > 12.0?

Reply:
Improves performance in Windows 10 but as I not tested on older systems prefer leave as before in these systems (Windows 7, 8)

Reply:
any proofs?

Reply:
Tested some months ago on Intel NUC8i3BEK: is needed play 4K HDR10 **60fps** content to appreciate difference.
6 back buffers is more important but these also helps to erradicate completly all frame drops all the time.